Sources/Paginator/Paginatable/RawSQLBuilderPaginatable.swift

```diff
@@ -0,0 +1,103 @@
+import Fluent
+import Vapor
+import SQL
+
+protocol RawSQLBuilderPaginatable: Paginatable {
+    associatedtype PaginatableMetaData
+    
+    static func paginate<D: Database, Result>(
+        source: RawSQLBuilder<D, Result>,
+        count: Future<Int>,
+        on req: Request
+        ) throws -> Future<([Result], PaginatableMetaData)>
+}
+
+class RawSQLBuilder<Database, Result> where Database: DatabaseKit.Database, Database.Connection: SQLConnectable, Result: Decodable {
+    let sqlRawBuilder: SQLRawBuilder<Database.Connection>
```

Comment:
What's the reason for these two being optionals?

Reply:
I left the count raw sql builder optional in order for the user to provide his own count producing  `Future`. I forgot to set the count parameter as optional for the `RawSQLBuilder` constructor. It's now done.